### ISSUE TITLE
Link component - you cannot trigger it using the RETURN key (Accessability)

### ISSUE THREAD

**Issue by UBOrange** [telekom/scale#605]
It should be possible (as a matter of accessability) to open a link by tabbing to the link and then pressing the "RETURN" key (not using the mouse, using the keyboard only).
Problem: It seems that you definitely need a mouse to follow a link (implemented using the "Link" component). "Keyboard only" users do not have access.

Note: The "Button" component works as expected.

**Comment by acstll** [comment]
This one is working for me in MS Edge. I can focus the link and hitting Return will trigger it. I just tested in Storybook, changing the `href` value in the Controls.

But neither Firefox nor Safari focus links by default in macOS. Could that be your case?

https://stackoverflow.com/a/11713537

![image](https://user-images.githubusercontent.com/851105/135601156-8831f765-390e-4a83-9040-375eb4a0464e.png)

Thanks again for the feedback!

**Comment by lstraka-telekom** [comment]
Hi @acstll 
 I found also one problem or issue with the ```<scale-link>```,

We would like to use link without href and without underline so ```<scale-link omit-underline> ``` is used and here is the issue that without href it looks like link have no possibility to gain focus with a keyboard. Allready checked in a docs and there is also not possible to focus with a keyboard. 

To be more specific, we need to make something like a expand/collapse "clickable link" but with just a icon and label no button or something different something without padding and border. Issue is that we need to make it focusable with the keyboard, but focus is not able to gain, only when used `href="#" ` inside it then it works, but page is refreshed and that\`s something what we do not want to achieve. I think everything should be fixed by using just a `tabindex="0" ` but it is not worked. And if `tabindex` is used for example over `<scale-card>` the focus of activeElement in that case had diferent color of focus/active (looks like some built in from a browsers, but yeah it`s another topic :D

Of course it is not a huge prio issue, we can skip it and replace it with a native `<span tabindex="0">` for example to have posibility to focus it but in that case again we are responsible to follow accesibility for a screen reader and other point related to accesibility and that is the highest benefit for us to keep accesibility in other hands of you  not on us.  😄

**Comment by acstll** [comment]
Hey @lstraka-telekom so sorry for the delayed reply, I missed it somehow.

A link without href is usually bad practice (it could even be invalid HTML). `<span tabindex="0" role="button">` could do it, but you don't have keyboard support.

I would suggest you use a button for this and set `aria-expanded` accordingly. Probably the `scale-collapsible` component (used in the Accordion component) would work for you?

Also, there's an undocumented variant for the button called `ghost` that might give you what you're looking for visually: `<scale-button variant="ghost">`. Otherwise you can create your own variant with some CSS and use it like `<scale-button class="my-variant">`:

```css
scale-button.my-variant::part(base) {
  /* custom styles here */
}
```

I hope this helps!

**Comment by lstraka-telekom** [comment]
Thank you, I will try ...I hope one of those proposal should be applied.  

Btw. I see there is a problem to sync official docs with all changes or new features provided, maybe, it should help to mentioned on Docs official page that, last update of docs was provided for release version XX. And in that case it should be much better to compare changes also in release notes if there were some changes provided for any component like <scale-button> etc... ? what do you think? I know that higer prio is always support customers with new a new features, like docs :D

**Comment by nowseemee** [comment]
I'm closing this for now but feel free to reopen whenever anyone of you cycles back to this topic.
